src/azul/indexer/index_controller.py

```diff
@@ -274,6 +292,20 @@ def aggregate(self, event: Iterable[SQSRecord], *, retry=False):
             log.warning('Failed to aggregate tallies: %r', tallies_by_entity.values(), exc_info=True)
             raise
 
+    def redrive(self, event):
+        for record in event:
+            message = json.loads(record.body)
+            receive_count = record.to_dict()['attributes']['ApproximateReceiveCount']
+            attempts = 1 if int(receive_count) > 1 else 0 + message['attempts']
+            log.info(f'Re-driver handling message {message}, attempt #{attempts} (approx).')
+            if message['attempts'] == 9:
```

Comment:
What is the significance of the number 9 here? If it's an important constant, it should be given a name and probably moved to `azul.Config`. I would also add an `assert` to fire when `message['attempts'] > 9` since it seems we should never observe this.